wca-regulations.md

```diff
@@ -91,6 +91,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 2i2b) The competitor must not interact with (e.g. operate, hold, wear) any active camera. Exception: the competitor may wear a camera mounted on their head, as long as it is out of their sight and it is clear that they are not interacting with it (apart from wearing it).
 - 2j) The WCA Delegate may disqualify a competitor from a specific event.
     - 2j1) If a competitor is disqualified from an event for any reason, they are not eligible for any more attempts in the event.
+        - 2j1a) The results of all remaining attempts in this event are recorded as DNF.
```

Comment:
This or the? :P We use the above, and this below.

Reply:
We rarely use "this" in the regulations, so "the event" sounds better for me.

Reply:
Is it still possible to suggest here? I would propose that this phrase was part of 2j1, inside ( ):

2j1) If a competitor is disqualified from an event for any reason, they are not eligible for any more attempts in the event (i.e. the results of all remaining attempts in this event are recorded as DNF).

Reply:
Do you have a reason for us? :-)

I think it's not "i.e.", otherwise we would not need this addition?
Also, it's quite an important regulation (considering that it was decided inconsistently in the past), so hiding it in parentheses might not be a good idea.

Reply:
The reason is that this seems like a continuation or an explanation of the previous sentence.

"they can't do more attempts, and those remaining attempts are marked as DNF"

I don't feel too strong about this...if kept separately, I would suggest to add "In this case" to the beginning of 2j1a